server/package.json

```
@@ -4,8 +4,9 @@
   "private": true,
   "scripts": {
     "start": "node app.js",
+    "dev": "nodemon server.ts -L --ext ts,json,js",
     "ts-build": "tsc",
-    "test": "mocha --exit --timeout 45000"
+    "test": "chmod +x ../scripts/wait-for-it.sh && ../scripts/wait-for-it.sh db:3306 --timeout=60 -- mocha --exit --timeout 45000"
```

I think the `"test"` command should still be the `mocha --exit --timeout` stuff, and this specifically should be `test-docker`:

```suggestion
    "test": "mocha --exit --timeout 45000",
    "test-docker": "chmod +x ../scripts/wait-for-it.sh && ../scripts/wait-for-it.sh db:3306 --timeout=60 -- npm run test"
```

That way we're separating out the actual test command from the docker entrypoint command.